pkg/controller/volume/attachdetach/reconciler/reconciler_test.go

```diff
@@ -455,13 +451,9 @@ func Test_Run_OneVolumeAttachAndDetachMultipleNodesWithReadWriteOnce(t *testing.
 	waitForTotalAttachCallCount(t, 1 /* expectedAttachCallCount */, fakePlugin)
 	verifyNewDetacherCallCount(t, true /* expectZeroNewDetacherCallCount */, fakePlugin)
 	waitForDetachCallCount(t, 0 /* expectedDetachCallCount */, fakePlugin)
+	waitForAttachedToNodesCount(t, 2, generatedVolumeName, asw)
```

Comment:
* Is `2` correct here? Previously the code was failing test if `if len(nodesForVolume) != 1`?
* nit: add parameter name as a comment, e.g. `/* expectedNodeCount */` next to parameter values that may not be immediately obvious.